test/unit/marked-spec.js

```diff
@@ -63,6 +63,12 @@ describe('Test slugger functionality', () => {
     const slugger = new marked.Slugger();
     expect(slugger.slug('<em>html</em>')).toBe('html');
   });
+
+  it('should not increment seen when just getting text', () => {
+    const slugger = new marked.Slugger();
+    slugger.slugText('<h1>This Section</h1>');
```

Comment:
we should test the output of the dryrun as well. Could you also add a test for a dryrun after a regular run?

Reply:
Of course 👍